src/pages/Transaction/utils.tsx

```diff
@@ -105,7 +106,7 @@ function getBalanceMap(transaction: Types.Transaction) {
       },
       event: Types.Event,
     ) => {
-      const addr = event.guid.account_address;
```

Comment:
So this is the only functional change, right?

Reply:
correct